workspace/src/path_planning/path_planning/path_planning.py

```diff
@@ -59,12 +59,21 @@ def __init__(self):
         package_share_directory = get_package_share_directory('path_planning')
 
         # READ IN PARAMETERS
+        self.declare_parameter('planning_mode', "cones")
+        self.planning_mode = self.get_parameter('planning_mode').get_parameter_value().string_value
+
         self.declare_parameter('vis', False)
         self.vis = self.get_parameter('vis').get_parameter_value().bool_value
 
         self.declare_parameter('lookahead', 2.0)
         self.lookahead = self.get_parameter('lookahead').get_parameter_value().double_value
         
+        if(self.planning_mode == "waypoints"):
+            self.declare_parameter('path_file', "/home/art/art/workspace/src/path_planning/path_planning/path.csv")
```

Comment:
@AaronYoung5 not sure what the best way to load this in is, as it is a csv file and not a yaml file of parameters. Any suggestions?

Reply:
I would use `launch_utils.utilities.GetPackageSharePath` and make a `data` folder in `path_planning`. So then like `launch_utils.utilities.GetPackageSharePath('path_planning') / `data` / <path file>`. `<path file>` should probs be a launch configuration. But anyways, ya probs best to put in launch. Make sure you add the `data` directory in the `CMakeLists.txt` file so that it gets installed.